datapackage.json

```diff
@@ -935,6 +950,15 @@
             }
         ],
         "entities": [
+            {
```

Comment:
Looks like you didn't update datapackage.json after removing `tag` entities again ; )

Reply:
Yup, and it worked perfectly, Thanks for the tip. I'll remove it, and update the datapackage.json.